src/notebooks/export/exportUtil.ts

```diff
@@ -3,6 +3,9 @@ import { NotebookCellData, NotebookData, NotebookDocument, Uri } from 'vscode';
 import { IExtensions } from '../../platform/common/types';
 import { ExportFormat, IExportDialog } from './types';
 
+/**
+ * Export utilities that are common to node/web

```

Comment:
E.g. this comment feels pointless to me,
no need to remove, but i don't think this would be useful
Having meaningful name and readable code makes this unnecessary (and the fact that the convention of having non .web and .node extension makes it obvious this is common for node & web)

Reply:
These comments also show up when hovering over usages of a class, so the contents and the file name may not be apparent when hovering over a use.

Reply:
Meaning they have more use than just opening this file and looking at the comment.

Reply:
Here's what I'm getting at
`ExportUtilsBase` is the name of the class.

* `Export` tells us its related to export
* `Utils` tells us that its a utility class
* The fact that it doesn't have a .web.ts nor a .node.ts extension means (by convention in this repo) its common to both web and node.

Hence the argument, this comment is not necessary nor does it add any new information that can be determined from the name of the class & file (which is already meaningfully named).

Again, no need to remove & possible others find this useful.

Reply:
Are you arguing we should skip documenting each class? Or maybe only document them if the description can offer something non obvious?

I find that the documentation comes up in hover useful (for this case it would be because the file name isn't obvious when just hovering over something)